packages/synapse-react-client/src/components/PortalFeatureHighlights/PortalFeatureHighlights.tsx

```diff
@@ -0,0 +1,86 @@
+import { Typography, Box, Button, CardMedia, Grid } from '@mui/material'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export type PortalFeatureHighlightsProps = {
+  reverseOrder?: boolean
+  title?: string
+  image?: string
+  buttonText?: string
+  summaryText?: string | React.ReactNode
```

Comment:
`string` is included in the `ReactNode` type
```suggestion
  summaryText?: React.ReactNode
```